Text/CustomEntitySearch/CustomEntitySearch.cs

```diff
@@ -50,38 +52,38 @@ public static class CustomEntitySearch
             }
 
             WebApiSkillResponse response = WebApiSkillHelpers.ProcessRequestRecords(skillName, requestRecords,
-                (inRecord, outRecord) => {
+                 (inRecord, outRecord) => {
                     string text = inRecord.Data["text"] as string;
                     IList<string> words = ((JArray)inRecord.Data["words"]).ToObject<List<string>>();
-                    if (words == null)
+                    Dictionary<string, string[]> synonyms = (inRecord.Data.ContainsKey("synonyms")) ? ((JContainer)inRecord.Data["synonyms"]).ToObject<Dictionary<string, string[]>>() : new Dictionary<string, string[]>();
+                    IList<string> exactMatches = (inRecord.Data.ContainsKey("exactMatches")) ? ((JArray)inRecord.Data["exactMatches"]).ToObject<List<string>>() : new List<string>();
+                    int offset = (inRecord.Data.ContainsKey("fuzzyMatchOffset") && (int)inRecord.Data["fuzzyMatchOffset"] >= 0) ? (int)inRecord.Data["fuzzyMatchOffset"] : 0;
+                    if (words.Count == 1 && words[0] == "")
                     {
-                        words = new WordLinker(executionContext.FunctionAppDirectory).Words;
+                        WordLinker userInput = new WordLinker(executionContext.FunctionAppDirectory);
```

Comment:
>WordLinker [](start = 51, length = 10)

Can you use this same model to deserialize the request payload too?

Reply:
Yes, but based on the existing method of using WebApiSkillResponse, I chose not to.